src/components/app/details/cdDetails/DeploymentDetailSteps.tsx

```diff
@@ -33,15 +35,24 @@ export default function DeploymentDetailSteps() {
     }
 
     useEffect(() => {
-        getDeploymentDetailStepsData()
-        return (): void => {
-            if (initTimer) {
-                clearTimeout(initTimer)
+        if (deploymentStatus !== 'Aborted') {
+            getDeploymentDetailStepsData()
+            return (): void => {
```

Comment:
Move this return outside if condition & add condition inside return, if required.